### PR TITLE
🐛 Source Intercom: Fix schema ts and array issue

### LINKED ISSUE #5882: Source Intercom Connector: inconsistency between schema and data
## Enviroment
- **Source Connector and version**: airbyte/source-intercom:0.1.2
- **Severity**: High 

## Current Behavior
Field is marked as String in Schema, but comes as Object in Data.

## Expected Behavior
Consistency between field and data.

## Logs
```
SCHEMA[
Field{name=id, type=STRING, mode=NULLABLE, description=null, policyTags=null},
Field{name=name, type=STRING, mode=NULLABLE, description=null, policyTags=null},
Field{name=plan, type=RECORD, mode=NULLABLE, description=null, policyTags=null},
Field{name=size, type=INTEGER, mode=NULLABLE, description=null, policyTags=null},
Field{name=tags, type=STRING, mode=null, description=null, policyTags=null},
Field{name=type, type=STRING, mode=NULLABLE, description=null, policyTags=null},
Field{name=app_id, type=STRING, mode=NULLABLE, description=null, policyTags=null},
Field{name=website, type=STRING, mode=NULLABLE, description=null, policyTags=null},
Field{name=industry, type=STRING, mode=NULLABLE, description=null, policyTags=null},
Field{name=segments, type=STRING, mode=null, description=null, policyTags=null},
Field{name=company_id, type=STRING, mode=NULLABLE, description=null, policyTags=null},
Field{name=created_at, type=INTEGER, mode=NULLABLE, description=null, policyTags=null},
Field{name=updated_at, type=STRING, mode=NULLABLE, description=null, policyTags=null},
Field{name=user_count, type=INTEGER, mode=NULLABLE, description=null, policyTags=null},
Field{name=monthly_spend, type=FLOAT, mode=NULLABLE, description=null, policyTags=null},
Field{name=session_count, type=INTEGER, mode=NULLABLE, description=null, policyTags=null},
Field{name=custom_attributes, type=RECORD, mode=NULLABLE, description=null, policyTags=null},
Field{name=remote_created_at, type=INTEGER, mode=NULLABLE, description=null, policyTags=null},
Field{name=_airbyte_ab_id, type=STRING, mode=null, description=null, policyTags=null}, Field{name=_airbyte_emitted_at, type=TIMESTAMP, mode=null, description=null, policyTags=null}]

DATA{
"type":"company",
"company_id":"5fcf68ed2022a598936dc693-qualification-company",
"id":"5fcf68ed2022a598936dc692",
"app_id":"wjw5eps7",
"name":"Another company",
"created_at":1607428333,
"updated_at":"2020-12-08T11:52:16",
"monthly_spend":0,
"session_count":0,
"user_count":1,
"size":700,
"tags":{"type":"tag.list","tags":[]},
"segments":{"type":"segment.list","segments":[]},
"plan":{},
"custom_attributes":{}}

```

## Steps to Reproduce

1. Setup intercom source
2. Setup Bigquery-denormalized destination
3. Create connection between above source/destination and run sync

### PR DESCRIPTION
**Pull request by lazebnyi**
## What
#5882 - Source Intercom Connector: inconsistency between schema and data

## How
Deleted `"format": "date-time"` for fields which type is integer.
Updated companies schema for tags and segments which content arrays.

## Recommended reading order
1. `x.java`
2. `y.python`

## Pre-merge Checklist
Expand the relevant checklist and delete the others. 

<details><summary> <strong> New Connector </strong></summary>
<p>
   
#### Community member or Airbyter
   
- [ ] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret` 
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated 
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] `docs/SUMMARY.md`
    - [ ] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [ ] `docs/integrations/README.md`
    - [ ] `airbyte-integrations/builds.md`
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] Connector added to connector index like described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
   
#### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items. 
   
- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] Credentials added to Github CI. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci). 
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing. 
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
   
</p>
</details>

<details><summary> <strong> Updating a connector </strong></summary>
<p>
   
#### Community member or Airbyter
   
- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret` 
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated 
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] Connector version bumped like described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
   
#### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items. 
   
- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] Credentials added to Github CI. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci). 
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing. 
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

</p>
</details>

<details><summary> <strong> Connector Generator </strong> </summary>
<p>
   
- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- [ ] The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates` then checking in your changes
- [ ] Documentation which references the generator is updated as needed.
</p>
</details>

### CODE REVIEW

**Comment by lazebnyi** [comment]
Find bug on the destination side will be created a follow-up ticket by java team.

**Review by bazarnov — APPROVED** [review]

**Review by antixar — APPROVED** [review]
Nice!

File: docs/integrations/sources/intercom.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ Please read [How to get your Access Token](https://developers.intercom.com/build
 
 | Version | Date       | Pull Request | Subject |
 | :------ | :--------  | :-----       | :------ |
+| 0.1.3   | 2021-09-08 | [0000](https://github.com/airbytehq/airbyte/pull/0000) | Corrected timestamp and arrays in schemas |
```
Comment (sherifnada) [inline docs/integrations/sources/intercom.md:58]:
```suggestion
| 0.1.3   | 2021-09-08 | [5908](https://github.com/airbytehq/airbyte/pull/0000) | Corrected timestamp and arrays in schemas |
```

**Review by sherifnada — APPROVED** [review]

**Pull request merged**